DEVELOP.rst

```diff
@@ -45,16 +45,24 @@ Preparing a Release
 
 To create a new release, you must:
 
+- Backport your changes to the release branch x.y
+
+In the release branch:
+
 - Update ``__version__`` in ``src/crate/client/__init__.py``
 
 - Add a section for the new version in the ``CHANGES.txt`` file
 
 - Commit your changes with a message like "prepare release x.y.z"
 
-- Push to origin
+- Push to origin/<release_branch>
 
 - Create a tag by running ``./devtools/create_tag.sh``
 
+On the master branch if the release was based on the master branch:
```

Comment:
I'd shorten this to:

    On master:

      - Update the release notes to reflect the release.


If this is done via cherry-pick, merge or even manually doesn't matter. The
result is what's important.